src/neoxp/Commands/ContractCommand.Download.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using McMaster.Extensions.CommandLineUtils;
+using Neo;
+using Neo.IO.Json;
+using Neo.Network.RPC;
+
+namespace NeoExpress.Commands
+{
+    partial class ContractCommand
+    {
+        [Command(Name = "download", Description = "Download contract with storage from remote chain into local chain")]
+        internal class Download
+        {
+            readonly ExpressChainManagerFactory chainManagerFactory;
+            private RpcClient rpcClient = null!;
+            
+            public Download(ExpressChainManagerFactory chainManagerFactory)
+            {
+                this.chainManagerFactory = chainManagerFactory;
+            }
+            
+            [Argument(0, Description = "Contract invocation hash")]
+            [Required]
+            internal string Contract { get; init; } = string.Empty;
+            
+            [Argument(1, Description = "Source network RPC address")]
+            internal string Source { get; init; } = string.Empty;
```

Comment:
The policy command also downloads content from a public chain. Policy command calls this argument `RpcUri`. We should be consistent. Also, the Policy command allows the user to specify "mainnet" or "testnet" in addition to supporting a full URL